frontend/src/ui/D3Visualizations/TreemapWrapper.tsx

```diff
@@ -53,11 +53,15 @@ export default function TreemapWrapper({
         selectedCategory={selectedCategory}
       />
       <CaptionText>
-        The treemap above shows the occupations which {selectedOccupation.name}{' '}
-        move to when they change occupation based on the observations in our
-        dataset. The transition share is the percentage of these observed{' '}
-        {selectedOccupation.name} who have moved into each of the occupations
-        listed.
+        This visualization shows the occupations which Sales Managers move to
```

Comment:
"Sales Managers" is hardcoded on lines 56 & 58.  It should be `{selectedOccupation.name}{' '}`

Reply:
all set.